terraform/modules/core/core.tf

```diff
@@ -398,14 +398,14 @@ resource "aws_autoscaling_policy" "ecs-autoscaling-down-policy" {
 }
 
 resource "aws_cloudwatch_metric_alarm" "average-reserved-cpu-high" {
-  alarm_name          = "${var.project_name}-${var.environment}-average-reserved-cpu-high"
+  alarm_name          = "${var.project_name}-${var.environment}-cluster-average-reserved-cpu-high"
   comparison_operator = "GreaterThanOrEqualToThreshold"
-  evaluation_periods  = "2"
+  evaluation_periods  = "1"
```

Comment:
Let's wait less time before scaling up instances. If EC2 instances have more than 40% CPU reserved for 60 seconds, we infer that ECS is adding new containers to respond to load and we should now add more EC2 instances to allow ECS to continue. We should scale up quickly, and scale down slowly.